src/main/java/seedu/address/logic/commands/csvcommand/LoadCommand.java

```diff
@@ -0,0 +1,173 @@
+package seedu.address.logic.commands.csvcommand;
+
+import static seedu.address.logic.parser.CliSyntax.PREFIX_FILE_NAME;
+
+import java.io.BufferedReader;
+import java.io.File;
+import java.io.FileReader;
+import java.io.IOException;
+
+import seedu.address.commons.exceptions.AlfredException;
+import seedu.address.commons.util.FileUtil;
+import seedu.address.logic.commands.Command;
+import seedu.address.logic.commands.CommandResult;
+import seedu.address.logic.commands.csvcommand.csvutil.CsvUtil;
+import seedu.address.logic.commands.csvcommand.csvutil.ErrorTracker;
+import seedu.address.logic.commands.csvcommand.csvutil.ErrorTracker.Error;
+import seedu.address.logic.commands.exceptions.CommandException;
+import seedu.address.model.Model;
+import seedu.address.model.entity.Entity;
+import seedu.address.model.entity.Mentor;
+import seedu.address.model.entity.Participant;
+import seedu.address.model.entity.Team;
+
+/**
+ * Supports bulk registration via a CSV file.
+ * This command aims to facilitate registration of entities onto Alfred.
+ */
+public class LoadCommand extends Command {
+
+    public static final String COMMAND_WORD = "load";
+
+    public static final String MESSAGE_SUCCESS = "Successfully loaded CSV file into Alfred";
+    public static final String MESSAGE_PARTIAL_SUCCESS = "Following line(s) were unable to be loaded into Alfred\n"
+             + "Possible reasons include incorrect formatting or adding of duplicate Entity:";
+    public static final String MESSAGE_FILE_NOT_FOUND = "File not found at %s"; // %s -> this.csvFileName
+    public static final String MESSAGE_IO_EXCEPTION = "An IOException was caught: %s"; // %s -> exception message
+    public static final String MESSAGE_INVALID_DATA = "CSV file contains invalid data";
+    public static final String MESSAGE_INVALID_FORMAT = "CSV file must contain Entity data in the following format:\n"
+            + "\tMentors: " + CsvUtil.HEADER_MENTOR + "\n"
+            + "\tParticipants: " + CsvUtil.HEADER_PARTICIPANT + "\n"
+            + "\tTeams: " + CsvUtil.HEADER_TEAM;
+    public static final String CAUSE_INVALID_DATA = "Invalid data format";
+    public static final String CAUSE_DUPLICATE_ENTITY = "This entity already exists in Alfred";
+    public static final String ASSERTION_FAILED_NOT_CSV = "File given is not a CSV file.";
+
+    public static final String MESSAGE_USAGE = COMMAND_WORD + ": Loads data in CSV file into Alfred"
+            + " Parameters: "
+            + PREFIX_FILE_NAME + "CSV_FILE_NAME\n"
+            + "\tExample (Windows): " + COMMAND_WORD
+            + " " + PREFIX_FILE_NAME + "C:/Users/USER/AlfredData/Alfred.csv\n"
+            + "Note the path does not have to be absolute (i.e. can be relative). "
```

Comment:
Same with this. I would suggest dumbing it down a little. A user may not know what absolute and relative mean in terms of paths.